### ISSUE TITLE
Vagrant Localhost auth breaks without valid xauth s3 key

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#444]
@bfalling, the OL development and production servers have a s3 key in their config for a special role user with access to our OL / IA xauthn bridge.

Just for background... When an OL users logs in, an ajax http request is made to an archive.org  service called "xauth" (cross authentication). This is what lets OL use and verify archive.org  credentials and link accounts between IA and OL.

This s3 key (mentioned above) is for a special privileged `role` archive.org user (identified by S3 keys) which is capable of making requests to the xauth archive.org service (and thus authenticating Open Library login)

TL;DR, without a valid s3 key to a sufficiently permissioned role user which can access archive.org's xauth service, our local vagrant instances will not be able to phone home to xauth and login the user.

**Solutions**

It looks like we're going to create a read-only developer key for each developer (i.e. no `create` privileges).

cc: @skylerbunny, @dvanduzer, @hornc

**Comment by brenton** [comment]
I'm assuming this issue was intended for someone else.

**Comment by mekarpeles** [comment]
Yes but feel free to chime in and help us create the world's best online public library :)

**Comment by dvanduzer** [comment]
Pull request in #445
